CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+# How to contribute
+
+In general the language for communication and documentation in this project is US English.
+
+## Commit messages
+
+Type a short, meaningful commit message that describes the change you made to the file.
+
+Please consider [the recommendation of the Git community for commit messages](https://git-scm.com/docs/git-commit/2.38.0#_discussion):
+
+> Begin the commit message with a single short (less than 50 character) line summarizing the change, followed by a blank line and then a more thorough description. The text up to the first blank line in a commit message is treated as the commit title, and that title is used throughout Git.
+
+## Pull requests
+
+Don't forget to self-review to speed up the review process.
+
+- Review the content for technical accuracy.
```

Comment:
Why are those items in a list and the above is not?

Reply:
Because I consider both items as part of the (self-)review.